docs/user/alerting/rule-types/es-query.asciidoc

```diff
@@ -143,44 +143,49 @@ You can further refine the conditions under which actions run by specifying that
 [float]
 === Add action variables
 
-You can pass rule values to an action to provide contextual details.
-To view the list of variables available for each action, click the "add rule variable" button.
+When you create a rule in {kib}, it provides an example message that is appropriate for each action.
+For example, the following message is provided for server log connector actions that run for each alert:
+
+[source,mustache]
+--------------------------------------------------
+Elasticsearch query rule '{{rule.name}}' is active:
+
+- Value: {{context.value}}
+- Conditions Met: {{context.conditions}} over {{rule.params.timeWindowSize}}{{rule.params.timeWindowUnit}}
+- Timestamp: {{context.date}}
+- Link: {{context.link}}
+--------------------------------------------------
+
+Rules use rule action variables and Mustache templates to pass contextual details into the alert notifications.
+There is a set of <<defining-rules-actions-variables,variables common to all rules>> and a set that is specific to this rule.
+To view the list of variables in {kib}, click the "add rule variable" button.
 For example:
 
 [role="screenshot"]
 image::images/es-query-rule-action-variables.png[Passing rule values to an action]
 // NOTE: This is an autogenerated screenshot. Do not edit it directly.
 
-The following variables are specific to the {es} query rule.
-You can also specify <<defining-rules-actions-variables,variables common to all rules>>.
-
-`context.title`:: A preconstructed title for the rule. Example: 
-`rule term match alert query matched`.
-
-`context.message`:: A preconstructed message for the rule. Example: +
-`rule 'my es-query' is active:` +
-`- Value: 2` +
-`- Conditions Met: Number of matching documents is greater than 1 over 5m` +
-`- Timestamp: 2022-02-03T20:29:27.732Z`
+The following variables are specific to the {es} query rule:
 
-`context.group`:: The name of the action group associated with the condition. 
-Example: `query matched`.
+`context.conditions`::
+(string) A description of the condition. For example: `Query matched documents`.
 
-`context.date`:: The date, in ISO format, that the rule met the condition. 
-Example: `2022-02-03T20:29:27.732Z`.
+`context.date`::
+(string) The date, in ISO format, that the rule met the condition. For example: `2024-04-30T00:55:42.765Z`.
 
-`context.value`:: The value of the rule that met the condition.
+`context.group`::
+(string) The name of the action group associated with the condition. 
+For example: `query matched`.
+//TBD: Is this still accurate? Missing from test output.
```

Comment:
I don't think we're setting this context variable so it can be removed

Reply:
Thanks! Removed in 2ad9423664fd1c5b5175def62283046b3702e4f4